.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,82 @@
 Hi! Before submitting your PR for review, and/or before merging it, please
-go through the following checklist:
+go through the following checklists:
 
-- [ ] For DB changes, check for missing indexes, check to see if the changes
-affect other apps (such as the dashboard), make sure the DB columns in the
-various environments are properly populated, coordinate with devops, plan
-migrations in separate steps.
+### General
 
-- [ ] For route changes, make sure GET requests don't change state or result in
-destructive behavior. GET requests should only result in information being
-read, not written.
+- [ ] `make lint` ran locally without errors
 
-- [ ] For encryption changes, make sure it is compatible with data that was
-encrypted with the old code.
+- [ ] Rubocop or Reek offenses were not disabled unless they are false positives.
+If you're not sure, please ask a teammate.
 
-- [ ] For secrets changes, [make sure to update the S3 secrets bucket](https://github.com/18F/identity-private/wiki/Secrets-S3-buckets) with the 
-new configs in **all** environments. 
+- [ ] Tests added for this feature/bug
 
-- [ ] Do not disable Rubocop or Reek offenses unless you are absolutely sure
-they are false positives. If you're not sure how to fix the offense, please
-ask a teammate.
+- [ ] The PR adheres to our [PR guidelines]
+
+- [ ] Related commits have been squashed
+
+- [ ] Method names in classes are in descending order of abstraction.
+See https://youtu.be/0rsilnpU1DU?t=554
+
+### Database
+
+- [ ] Unsafe migrations are implemented over several PRs and over several
+deploys to avoid production errors. The `strong_migrations` gem
+will warn you about unsafe migrations. Please follow the gem's advice.
+
+- [ ] Indexes were added if necessary
+
+- [ ] Verified that the changes don't affect other apps (such as the dashboard)
+
+- [ ] When relevant, an issue has been opened as a reminder to populate the
```

Comment:
It would be nice if we created a rake task for populating the columns that could be run. Not required, and I've been places where we created a module with a method that we could invoke from the rails console, which is just about as good if it's not a complicated invocation.